src/book.js

```diff
@@ -139,11 +139,17 @@ export class Book {
     this.inputpdf = file.name;
     this.input = await file.arrayBuffer(); //fs.readFileSync(filepath);
     this.currentdoc = await PDFDocument.load(this.input);
-    //TODO: handle pw-protected PDFs
+    this.fixBlankPages();
+  }
+
+  /**
+   * Modifies pages with no content to have an invisible line on them.
+   * Dumb hack because you can't embed blank pdf pages without errors.
+   */
+  fixBlankPages() {
```

Comment:
Love the break here

--my unasked for $0.02 on coding styles -- would love if in the future we operated on passed in values/returned things rather than reference member variables.

Reply:
I'm not entirely opposed, but stuff like that should probably be in an outside file, and we have to be careful not to end up with too many pass-through chains - part of the point of classes is to encapsulate state stuff like that.